File: Listas de exercícios em python/Lista 2 - Casa/Exe 2.py
numero = int(input("Digite um número inteiro (0 a 100): "))
NumeroChave = 35

if 0 <= numero <= 100:
    distancia = abs(NumeroChave-numero)
    print(f"Distancia entre {numero} e {NumeroChave} é {distancia}.")
else:
    print("Número fora dos limites! tente novamente (apenas entre 0 a 100).")
    

File: Listas de exercícios em C/Lista 2 - Casa/Exe 2.c
#include <stdio.h>
#include <stdlib.h> 

int main() {
    int numero;
    int NumeroChave = 35;

    printf("Digite um número inteiro (0 a 100): ");
    scanf("%d", &numero);

    if (numero >= 0 && numero <= 100) {

        int distancia = abs(NumeroChave - numero);

        printf("Distância entre %d e %d é %d.\n", numero, NumeroChave, distancia);
    } else {
       
        printf("Número fora dos limites! Tente novamente (apenas entre 0 a 100).\n");
    }

    return 0;
}